app/feedme/zendesk_email_parser.py

```
@@ -0,0 +1,775 @@
+            '%Y-%m-%dT%H:%M:%S%z',
+            '%Y-%m-%dT%H:%M:%SZ',
+            '%Y-%m-%d %H:%M:%S',
+            '%m/%d/%Y %I:%M %p',
+            '%d %b %Y %H:%M',
+            '%B %d, %Y at %I:%M %p',
+        ]
+        
+        for fmt in formats:
+            try:
+                return datetime.strptime(datetime_str, fmt)
+            except:
```

Ruff (0.11.9)

Do not use bare `except`

(E722)